.github/workflows/ci.yml

```
@@ -37,30 +43,31 @@ jobs:
     name: Go Test
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Go Test
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2094:info:2:26: Make sure not to read and write the same file in the same pipeline

(shellcheck)